apps/condo/domains/user/schema/RegisterNewUserService.js

```diff
@@ -85,27 +84,38 @@ const RegisterNewUserService = new GQLCustomSchema('RegisterNewUserService', {
                     const msg = '[error] Unable to create user'
                     throw new Error(msg)
                 }
-                // end
+                // end of TODO
                 if (confirmPhoneActionToken) {
                     await ConfirmPhoneActionServerUtils.update(context, confirmPhoneActionId, { completedAt: new Date().toISOString() })
                 }
+                const sendChannels = [{
+                    to: { phone: userData.phone },
+                }]
+                if (!isEmpty(userData.email)) {
+                    sendChannels.push({
+                        to: { email: userData.email },
+                    })
+                }
                 // TODO(Dimitreee): use locale from .env
                 const lang = COUNTRIES[RUSSIA_COUNTRY].locale
-                await sendMessage(context, {
-                    lang,
-                    to: {
-                        user: {
-                            id: user.id,
+                await Promise.all(sendChannels.map(async channel => {
+                    await sendMessage(context, {
+                        lang,
+                        to: {
+                            user: {
+                                id: user.id,
+                            },
+                            ...channel.to,
                         },
-                    },
-                    type: REGISTER_NEW_USER_MESSAGE_TYPE,
-                    meta: {
-                        userPassword: userData.password,
-                        userPhone: userData.phone,
-                        dv: 1,
-                    },
-                    sender: data.sender,
-                })
+                        type: REGISTER_NEW_USER_MESSAGE_TYPE,
```

Comment:
It's better to move select channel login inside the `sendMessage`. It's better to hide `channel` inside `sendMessage` because we can dynamically add a new channels

Reply:
We need to send the same message through email (if it presents) and phone. And the logic of choosing transport is already in sendMessage.
If we pass the email field to the message - email transport will be used
If we pass phone field to the message - SMS transport will be chosen
if we do not pass any of these fields to message the logic will be chosen according to user settings(in the future) or according to if email presents in the user record or not.